servers/omnisharp/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:bullseye-slim
+
+WORKDIR /tmp
+
+RUN apt update \
```

Comment:
Update this line from `apt` to `apt-get` to fix linting errors: https://github.com/lspcontainers/dockerfiles/runs/3526706197#step:3:13

Reply:
Changed apt to apt-get as suggested.

Reply:
Awesome - thank you! Got one more line right under - needs all instances updated.